.travis.yml

```diff
@@ -1,16 +1,47 @@
 language: python
-python: 2.7
+python: 
+  - 2.7
+  - 3.5
 addons:
   firefox: 47.0.1
-env:
-- TOXENV=tests DISPLAY=:99.0
-- TOXENV=flake8
+matrix:
+  include:
+    - python: 2.7
+      env: 
+        - TOXENV=flake8
+    - python: 2.7
+      env: 
+        - DISPLAY=:99.0
+        - TOXENV=py27
+    - python: 2.7
+      env: 
+        - DISPLAY=:99.0
+        - TOXENV=py27-marionette
```

Comment:
You shouldn't need the `py27-` prefix here, that's what `tox-travis` takes care of.